test/rule.js

```diff
@@ -0,0 +1,61 @@
+const PushStateTree = require('../src/pushStateTree');
```

Comment:
most of the tests in this file uses the createRule function which is tested in route-CreateRules.js.